model/registry/k8s.yaml

```diff
@@ -98,7 +98,12 @@ groups:
           Number of times the container was restarted. This attribute can be
           used to identify a particular container (running or stopped) within a
           container spec.
-        examples: [0, 2]
+      - id: container.status.last_terminated_reason
+        type: string
+        stability: experimental
+        brief: >
+          Last terminated reason of the Container.
+        examples: ["Evicted", "Error"]
```

Comment:
nit: Should we specify an enum with the possible values if there is such a list?

Reply:
My point is that if we know the possible values this attribute can take we can specify them explicitly like what we do in `cpu.state`s for example: https://github.com/open-telemetry/semantic-conventions/blob/4d80b4ce2b17c84856f39c73cef41ae54d087d32/model/registry/system.yaml#L19-L43

Reply:
I see, thank's for clarifying!